tests/python/rest_api/test_webhooks_sender.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2022 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import json
+from http import HTTPStatus
+
+import pytest
+from deepdiff import DeepDiff
+
+from shared.fixtures.init import _run
+from shared.utils.config import get_method, patch_method, post_method
+from webhook_receiver.server import PAYLOAD_ENDPOINT, TARGET_PORT
+
+# Testing webhook functionality:
+#  - webhook_receiver container receive post request and return responses with the same body
+#  - cvat save response body for each delivery
+#
+# So idea of this testing system is quite simple:
+#  1) trigger some webhook
+#  2) check that webhook is sent by checking value of `response` field for the last delivery of this webhook
+
+
+def webhook_container_id():
+    return _run(
+        "docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' test_webhook_receiver_1"
+    )[0].strip()[1:-1]
+
+
+def webhook_spec(events, project_id=None, webhook_type="organization"):
+    # Django URL field doesn't allow to use http://webhook:2020/payload (using alias)
+    # So we forced to use ip address of webhook receiver container
+    return {
+        "target_url": f"http://{webhook_container_id()}:{TARGET_PORT}/{PAYLOAD_ENDPOINT}",
```

Comment:
Need to try, but probably you can replace `{webhook_container_id}` by `webhooks` because it is the alias for the container and the contaner in the same network, as other containers are.

Reply:
Django return an error when I try to create webhook with `target_url` field that equals `http://webhooks:2020/payload`, even though `curl webhooks:2020` from cvat_server container works fine.